.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: elixir
-elixir: 1.4.2
-otp_release: 19.3
+elixir: 1.13.2
+otp_release: 24.2
 sudo: false
 matrix:
   include:
-    - elixir: 1.4.5
-      otp_release: 19.3
-    - elixir: 1.5.3
+    - elixir: 1.10.4
       otp_release: 20.3
-    - elixir: 1.6.6
-      otp_release: 20.3
-    - elixir: 1.7.3
-      otp_release: 21.0
+    - elixir: 1.11.4
+      otp_release: 24.2
+    - elixir: 1.12.3
+      otp_release: 24.0
+    - elixir: 1.13.2
+      otp_release: 24.2
```

Comment:
Cowboy 2.9 requires OTP 20+

Reply:
We upgrade Elixir and OTP versions only for the test environment. However, since CI no longer runs for old versions of Elixir, we may potentially break the package on old versions of Elixir without being noticed by the maintainer.

However, the trade-off is worthy IMO, because we have to move forward and prior new versions of Elixir and/or OTP.

Reply:
I don't actually know what Elixir's version support policy is.

So, I'm willing to drop official support for earlier versions if the tests don't compile.

In theory it should work as long as `proc_lib` and the general interface for `gen_tcp` doesn't change.

The thing I really worry about is Erlang's `ssl` module. But since we're essentially feeding options straight to Erlang `ssl` and doing very little in the library, I think that it's okay to presume that breaking changes in the `ssl` module must be handled by the user.
The interface _shouldn't_ end up differing from `gen_tcp`.

Sorry, I'm just thinking in text. I think it's the right choice to drop older versions.